README.rst

```diff
@@ -167,6 +167,12 @@ Links
 * `License <https://borgbackup.readthedocs.org/en/stable/authors.html#license>`_
 * `Security contact <https://borgbackup.readthedocs.io/en/latest/support.html#security-contact>`_
 
+Development
+-------------------
+Borg is written in Python. To get started with Borg development (including guidance for 
+contributions), please see development.rst: 
+https://github.com/borgbackup/borg/blob/master/docs/development.rst
+
```

Comment:
Thanks for your PR!

Line 149 talks about docs and development, so adding this section after line 169 seems to be a better place.

Also:
- please have length of the dashes line match the length of the headline text.
- "please see: https://..." (do not mention the filename, it is part of the URL anyway)

Hmm, also consider linking to the rendered docs rather than to the rst source files:

https://borgbackup.readthedocs.io/en/stable/development.html